### PR TITLE
Brads Random updates

### PR DESCRIPTION
**Pull request by bnmnetp**
I'm not sure these are exactly the right questions but the iteration chapter seems really light on Exercises.  We introduce the accumulator pattern but there are not really any exercises in the chapter to provide practice

### CODE REVIEW

File: _sources/Iteration/Exercises.rst [hunk 1]
```diff
@@ -14,31 +14,139 @@
...
+.. question:: iter_ex_7
+
+   .. tabbed:: q7
+
+       .. tab:: Question
+
+            .. actex:: ex_3_7
+               :nocodelens:
+
+               A drunk pirate makes a random turn and then takes 100 steps forward, makes another random turn, takes another 100 steps, turns another random amount, etc.  A social science student records the angle of each turn before the next 100 steps are taken.  Her experimental data is ``160, -43, 270, -97, -43, 200, -940, 17, -86``. (Positive angles are counter-clockwise.)  Use a turtle to draw the path taken by our drunk friend.  After the pirate is done walking, print the current heading.
```
Comment (presnick) [inline _sources/Iteration/Exercises.rst]: Add hints? Try to make it work for any sequence of experimental data. Iteration and the accumulator pattern might help you.

File: _sources/Iteration/Exercises.rst [hunk 2]
```diff
@@ -14,31 +14,139 @@
...
+               lovelace.pendown()
+               for angle in [160, -43, 270, -97, -43, 200, -940, 17, -86]:
+
+                   # we use .left() so that positive angles are counter-clockwise
+                   # and negative angles are clockwise
+                   lovelace.left(angle)
+                   lovelace.forward(100)
+
+               # the .heading() method gives us the turtle's current heading in degrees
+               print("The pirate's final heading was", lovelace.heading())
```
Comment (presnick) [inline _sources/Iteration/Exercises.rst]: Oh, I didn't realize there was a heading() method. I'm not sure we ever told them about that. How about adding a hint that the initial heading is <whatever it is> and then have them accumulate all the angle changes?

Comment (bnmnetp) [reply _sources/Iteration/Exercises.rst]: Sure having them accumulate the heading changes would be good.

Comment (bnmnetp) [reply _sources/Iteration/Exercises.rst]: As a bonus they will need to use the modulo operator to have the angle between 0-360

**Pull request merged**